XcodeServerSDK.xcodeproj/project.xcworkspace/xcshareddata/XcodeServerSDK.xccheckout

```diff
@@ -5,13 +5,21 @@
 	<key>IDESourceControlProjectFavoriteDictionaryKey</key>
 	<false/>
 	<key>IDESourceControlProjectIdentifier</key>
+<<<<<<< HEAD
```

Comment:
Actually, probably revert your file, I don't think we want to change the checkout file this way (it would change the origin URL etc)